schemas/bill_summaries.py

```diff
@@ -0,0 +1,26 @@
+from pydantic import BaseModel, Field
+
+
+class BillSummaryUsSectionSchema(BaseModel):
+    """'bill_summary_us' section schema."""
+
+    text: str | None = Field(None)
```

Comment:
instead of `str | None` try to use `Optional[str]`

IRC there was an issue in pydantic with such type hints. 
So we can simplify this model to 
```suggestion
    text: Optional[str]
```

Reply:
Since we are using python 3.10 + https://fastapi.tiangolo.com/tutorial/body-multiple-params/#multiple-body-params-and-query I think we should using newer syntax here